src/main/java/org/broadinstitute/hellbender/utils/HttpUtils.java

```diff
@@ -8,39 +8,40 @@
 import org.apache.http.protocol.HttpContext;
 
 public class HttpUtils {
-    private static CloseableHttpClient client;
+    private static CloseableHttpClient client = HttpUtils.makeClient();
 
     private HttpUtils() {}
 
-    public static synchronized CloseableHttpClient getClient() {
-        if (HttpUtils.client == null) {
-            HttpUtils.client = HttpClientBuilder.create()
-                .setConnectionManager(new PoolingHttpClientConnectionManager())
-                .setServiceUnavailableRetryStrategy(new ServiceUnavailableRetryStrategy() {
-                    private int interval = 1;
+    public static CloseableHttpClient getClient() {
+        return HttpUtils.client;
+    }
 
-                    @Override
-                    // retry at most 4 times if a 5xx status code is received, or no status line is present
-                    public boolean retryRequest(final HttpResponse resp, final int executionCount, final HttpContext context) {
-                        if (executionCount > 4) {
-                            return false;
-                        }
-                        if (resp.getStatusLine() == null) {
-                            return true;
-                        }
-                        final int statusCode = resp.getStatusLine().getStatusCode();
-                        return 500 <= statusCode && statusCode < 600;
-                    }
+    private static CloseableHttpClient makeClient() {
+        return HttpUtils.client = HttpClientBuilder.create()
+            .setConnectionManager(new PoolingHttpClientConnectionManager())
+            .setServiceUnavailableRetryStrategy(new ServiceUnavailableRetryStrategy() {
+                private int interval = 1;
 
-                    @Override
-                    public long getRetryInterval() {
-                        final int retryInterval = interval;
-                        interval *= 2;
-                        return retryInterval;
+                @Override
+                // retry at most 4 times if a 5xx status code is received, or no status line is present
```

Comment:
Does something get logged when retries happen / run out?

Reply:
It will just return the response that it got back, whatever it is.